frontend/src/Metamaps/DataModel/Mapping.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash'
+import Backbone from 'backbone'
+try { Backbone.$ = window.$ } catch (err) {}
+
+import Active from '../Active'
+import Map from '../Map'
+import Synapse from '../Synapse'
+import Topic from '../Topic'
+
+const Mapping = Backbone.Model.extend({
+  urlRoot: '/mappings',
+  blacklist: ['created_at', 'updated_at'],
+  toJSON: function (options) {
+    return _.omit(this.attributes, this.blacklist)
+  },
+  initialize: function () {
+    if (this.isNew()) {
+      this.set({
+        'user_id': Active.Mapper.id,
+        'map_id': Active.Map ? Active.Map.id : null
+      })
+    }
+  },
+  getMap: function () {
+    return Map.get(this.get('map_id'))
```

Comment:
I should convert these to promises, or callback style, if we want to keep these accessor functions for relations between models.